include/psp2/appmgr.h

```diff
@@ -67,11 +67,15 @@ typedef struct SceAppMgrSystemEvent {
 
 typedef struct SceAppMgrAppState SceAppMgrAppState; // Missing struct
 typedef struct SceAppMgrExecOptParam SceAppMgrExecOptParam; // Missing struct
+typedef struct SceAppMgrLaunchAppOptParam SceAppMgrLaunchAppOptParam; // Missing struct
 
 #define SCE_APPMGR_MAX_APP_NAME_LENGTH	(31)
 
 //! name: The Title ID of the application
-int _sceAppMgrDestroyAppByName(char *name);
+int sceAppMgrDestroyAppByName(char *name);
+
+//! name: The Title ID of the application
+int _sceAppMgrDestroyAppByName(char *name, char temp[3]);
```

Comment:
I think for consistency we should keep the SDK using the non-underscore API. It gives us two benefits:
- Consistency within the SDK
- Theoretical API protection. I assume that the underscore API is private/hidden from developers and thus can be changed. This of course, may be completely false.

This would be effectively reverting https://github.com/vitasdk/vita-headers/pull/46 which can break homebrew other than VitaShell. @xyzz @joshdekock any input?